extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/dialog.addmodule.html

```diff
@@ -7,14 +7,14 @@ <h1 ng-show="step==1">Select {{type}} type</h1>
 
 	<form name="myForm">
 
-		<md-input-container class="col-md-12 col-sm-12 configContainer" ng-show="step==2"> <label>Name</label> <input ng-model="name"> </md-input-container>
+		<md-input-container class="col-md-12 col-sm-12 col-xs-12 moduleContainer" ng-show="step==2"> <label>Name</label> <input ng-model="name"> </md-input-container>
```

Comment:
@watou Does this answer your question or did you actually want to say that something should be changed here?